kbase.yml

```diff
@@ -8,7 +8,7 @@ service-language:
     python
 
 module-version:
-    1.2.1
+    1.2.2
 
 owners:
     [dylan]
```

Comment:
Looks like James also did some work on this, should we add him in this list as well?

Reply:
I'm a catalog admin so I don't really need to be added to the owners unless you want it done for bookkeeping